pantry-finder-api/spec/serializers/foodbanks_serializer_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'byebug'
+
+describe FoodbankSerializer do
+  it 'creates JSON for single address field display_url is ""' do
+    serializer = FoodbankSerializer.new \
+      Foodbank.new(fb_name: 'local foodbank', address1: 'addr 1',
+                   city: 'the town', state: 'OH', zip: '12345')
```

Comment:
```suggestion
    serializer = FoodbankSerializer.new(
      Foodbank.new(fb_name: 'local foodbank', address1: 'addr 1',
                   city: 'the town', state: 'OH', zip: '12345')
    )
```
Seems more readable to me than the split with `\`. I had to think about it to understand that the foodbank on line 8 was an argument for `new`.